YAXLib/ISerializationContext.cs

```diff
@@ -1,49 +0,0 @@
-﻿// Copyright (C) Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors.
```

Comment:
Moved to folder "Custom". Maybe there's a better name so summarize the content of the folder.
Namespace of files in there is still `YAXLib`-